Code-the-Dream-School/bb-practicum-team-3-front
# integrated backend API into Sign in and Signup forms

## Description

Sign up
Sent a post request to server for "/register" endpoint, redirect a user to Home page if registration is successful 
Sign in
send a post request to server, endpoint "/login", if successful, redirect to Home page 

## Acceptance Criteria

Make the signup route functional with backend auth
Once you receive the response 200 direct the user to the Home page
When Login is successful direct the user to the Home page

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |


## Testing Steps / QA Criteria

start the back-end server first 
open “bb-practicum-bb-front-3” 
run npm start to start the development server
open http://localhost:3000/ with your browser
click “Sign up” button in Header section
enter valid credentials (password should be at least 6 characters long) 
click on the 'Sign up' button (if successful, it should direct the user to the Home page)
click “Sign in”  button in Header section 
enter valid credentials  (if successful, it should direct the user to the Home page)
